hs-bindgen/examples/anonymous.h

```diff
@@ -0,0 +1,9 @@
+// Struct containing an anonymous inner struct
```

Comment:
Not added to test-suite on purpose?

Reply:
Yeah, I guess I could. Right now they are mostly tiny examples that I'm using for ad-hoc tests for the additional bindings. Probably actually that `anonymous` thing should just be yet another example in the example structs, with a separate file for testing all the Doxygen stuff.